### PR TITLE
Feature/pie de pagina

### PR DESCRIPTION
**Pull request by clmtest**
# FooCamp - Escuela de Robotica

## Trello ticket
[Ticket Link](https://trello.com/c/lsLgPpW6/12-user-story-10-pie-de-p%C3%A1gina)

## Description
This PR contains the HTML and SCSS files for the pie-de-pagina of the Escuela de Robotica 

## To reproduce
1. Run the project with the following command line: npm run dev
2. Open the following page: http://localhost:8080/christian-page.html
3. Check the component

## Screenshots
Footer

### CODE REVIEW

File: src/scss/layout/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
+.footer{
+    background-color: #577598;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:2]: Please make this color part of the variables on the sass files

File: src/scss/layout/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    display: grid;
+    grid-template-columns: 2fr 1fr 1fr;
+    height: 461px;
+    max-width: 1440px;
+    margin: auto;
+
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:8]: Remove one of the extra spaces here

File: src/scss/layout/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    
+    &__section{
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__section--row{
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:15]: This query can be inside the `&__section` one.
You can do something like
```
&__section {
   &--row {
   
   }
}
```

File: src/scss/layout/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+    &__title{
+        width: 218px;
+        height: 108px;
+        font-family: Helvetica;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 30px;
+        line-height: 34px;
+        color: #FFFFFF;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:35]: color on the variable file

File: src/scss/layout/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
+.footer{
+    background-color: #577598;
+    display: grid;
+    grid-template-columns: 2fr 1fr 1fr;
+    height: 461px;
+    max-width: 1440px;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:6]: Please define this on a higher level. The max width of the footer is not only on this component, but the whole application

File: src/scss/layout/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+    &__paragraph{
+        width: 302px;
+        height: 117px;
+        font-family: Helvetica;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 16px;
+        line-height: 18px;
+        color: #FFFFFF;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:55]: color in the variable file

File: src/scss/layout/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+        font-weight: bold;
+    }
+
+    &__subtitles{
+        font-family: Helvetica;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 18px;
+        line-height: 24px;
+        color: #FFFFFF;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:68]: color in the variable file

File: src/scss/layout/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    &__links{
+        padding: 5px 3px;
+        font-family: Helvetica;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 16px;
+        color: #FFFFFF;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:79]: color in the variable file

File: src/scss/layout/_footer.scss [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+        height: 60px;
+        background: #F6BF42;
+        border-radius: 60px;
+        font-family: Helvetica;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 24px;
+        line-height: 28px;
+        text-align: center;
+        color: #2C8EFF;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:94]: color in the variable file

File: src/scss/layout/_footer.scss [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+        border-style: none;
+        margin-bottom: 44px;
+    }
+
+    &__image{
+        width: 26.05px;
```
Comment (camilomejiag) [inline src/scss/layout/_footer.scss:100]: Some browsers don't recognize decimal on pixels. Please keep them as integers.

File: templates/footer.hbs [hunk 11]
```diff
@@ -0,0 +1,58 @@
+    <footer class="footer">
+            <div>
```
Comment (camilomejiag) [inline templates/footer.hbs:2]: Give this div a class name. A div by itself is not very explaining

**Review by camilomejiag — CHANGES_REQUESTED** [review]

**Review by diegoamt — CHANGES_REQUESTED** [review]
The Footer should be always included in the page instead of adding it as a component in the Components section of the Page Model.
This means we should need to add the Footer on 'page.hbs' after adding components: `{{{module footer}}}`

**Review by camilomejiag — APPROVED** [review]

**Review by diegoamt — APPROVED** [review]

**Pull request merged**